hardware/pico_driver/scripts/pico_split_extended.py

```diff
@@ -98,25 +123,57 @@ def main():
         choices=pico.CAM_CHOICES,
         default=pico.DEFAULT_CAM,
     )
-    parser.add_argument("in_bag", help="input bag containing extended messages")
-    parser.add_argument("in_npy", help="input xyz coefficients file")
-    parser.add_argument("out_bag", help="output bag of points messages")
+    parser.add_argument(
+        "-s",
+        "--save-all-topics",
+        dest="save_all_topics",
+        action="store_true",
+        help="Save all topics from input bagfile to output bagfile.",
+    )
+    parser.add_argument(
+        "-k",
+        "--keep-extended",
+        dest="keep_extended_topic",
+        action="store_true",
+        help="Save extended topic alongside converted on the new bagfile",
+    )
+    parser.add_argument(
+        "-n",
+        dest="do_nothing",
+        action="store_true",
+        help="Option to not debayer anything and write output",
```

Comment:
I get the idea but I think this script is not related to debayering so it can be expressed better.